deploy.sh

```diff
@@ -16,14 +16,14 @@ ORG=${ORG:-gentoo}
 docker push --all-tags "${ORG}/${NAME}"
 
 declare -A MANIFEST_TAGS=(
-	[stage3:latest]="amd64-openrc;armv5tel;armv6j_hardfp;armv7a_hardfp;arm64;i686-openrc;ppc64le-openrc;s390x"
+	[stage3:latest]="amd64-openrc;armv5tel;armv6j_hardfp;armv7a_hardfp;arm64;i686-openrc;ppc64le-openrc;rv64_lp64-openrc;rv64_lp64d-openrc;s390x"
```

Comment:
Only a single manifest can represent the `linux/riscv64` Docker arch; pick one.

Reply:
Pick lp64d. lp64d is more popular in general.